src/main/java/pl/wojciechkostecki/revel/model/dto/MenuDTO.java

```diff
@@ -0,0 +1,21 @@
+package pl.wojciechkostecki.revel.model.dto;
+
+import lombok.Data;
+import pl.wojciechkostecki.revel.model.Local;
+import pl.wojciechkostecki.revel.model.MenuItem;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Data
+public class MenuDTO {
+    private Long id;
+
+    private String name;
+
+    private Local local;
```

Comment:
Why do you have Local and localId ? They both point to the same object. Do you even use the `local` field?

Reply:
I don't use local field anymore. After adding localId, I forgot to delete local field.